libs/generate-ui/feature-task-execution-form/src/lib/task-execution-form/task-execution-form.component.html

```diff
@@ -52,96 +52,179 @@
+                    <path
+                      fill-rule="evenodd"
+                      clip-rule="evenodd"
+                      d="M7 13.992H4v-9h8v2h1v-2.5l-.5-.5H11v-1h-1a2 2 0 0 0-4 0H4.94v1H3.5l-.5.5v10l.5.5H7v-1zm0-11.2a1 1 0 0 1 .8-.8 1 1 0 0 1 .58.06.94.94 0 0 1 .45.36 1 1 0 1 1-1.75.94 1 1 0 0 1-.08-.56zm7.08 9.46L13 13.342v-5.35h-1v5.34l-1.08-1.08-.71.71 1.94 1.93h.71l1.93-1.93-.71-.71zm-5.92-4.16h.71l1.93 1.93-.71.71-1.08-1.08v5.34h-1v-5.35l-1.08 1.09-.71-.71 1.94-1.93z"
+                    />
+                  </svg>
+                </button>
+              </div>
               <div
-                class="filter-flags-message"
-                *ngIf="filteredFields.size === 0"
-                #noMatch
+                class="details-link"
+                *ngIf="ide === 'vscode' && getDocsLink(taskExecForm)"
               >
+                <a [href]="getDocsLink(taskExecForm)"> View full details </a>
+              </div>
+            </div>
+            <div class="fx-row-reverse">
+              <nx-console-argument-list
+                [args]="runCommandArguments$ | async"
+              ></nx-console-argument-list>
+              <div class="fields-container">
+                <ng-container *ngIf="defaultValues$ | async as defaultValues">
+                  <nx-console-field
+                    [id]="field.name + '-nx-console-field'"
+                    *ngFor="let field of taskExecForm.architect.options"
+                    [class.changed-from-default]="
+                      defaultValues[field.name] !==
+                      taskExecForm.form.value[field.name]
+                    "
+                    [field]="field"
+                    [formControlName]="field.name"
+                    [style.display]="
+                      filteredFields.has(field.name) &&
+                      splitFields.important.has(field.name)
+                        ? 'block'
+                        : 'none'
+                    "
+                  ></nx-console-field>
+                  <div
+                    *ngIf="
+                      hasFilteredOtherFields(filteredFields, splitFields.other)
+                    "
+                  >
+                    <div
+                      *ngIf="
+                        hasFilteredImportantFields(
+                          filteredFields,
+                          splitFields.important
+                        )
+                      "
+                      (click)="toggleShowFields()"
+                      class="accordion-button"
+                    >
+                      <hr class="divider" />
+                      <div class="flexcontainer">
+                        <button class="options-button">
+                          {{
+                            showOtherFields
+                              ? 'Show less options'
+                              : 'Show all options'
+                          }}
+                        </button>
```

Comment:
Can we have this in one line? 
If necessary change the text, but a 2 lines button looks weird.